src/Functions/randomPrintableASCII.cpp

```diff
@@ -120,7 +120,30 @@ class FunctionRandomPrintableASCII : public IFunction
 
 REGISTER_FUNCTION(RandomPrintableASCII)
 {
-    factory.registerFunction<FunctionRandomPrintableASCII>();
+    FunctionDocumentation::Description description = R"(
+Generates a string with a random set of [ASCII](https://en.wikipedia.org/wiki/ASCII#Printable_characters) characters.
+All characters are printable.
+If you pass `length < 0`, the behavior of the function is undefined.
+    )";
+    FunctionDocumentation::Syntax syntax = "randomPrintableASCII(length)";
+    FunctionDocumentation::Arguments arguments = {
+        {"length", "String length in bytes.", {"(U)Int*", "Float*"}}
+    };
+    FunctionDocumentation::ReturnedValue returned_value = {"Returns a string with a random set of [ASCII](https://en.wikipedia.org/wiki/ASCII#Printable_characters) printable characters.", {"String"}};
```

Comment:
I think we don't need to link to Wikipedia here again (we did that in the description already).